### PR TITLE
a15-13638 Add Shipping options tests

### PR DESCRIPTION
**Pull request by danggrianto**

### CODE REVIEW

File: fixtures/cart.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 CART = {
-    "siteId": "string",
-    "language": "string",
+    "siteId": "uo-us",
```
Comment (OSeraph) [inline fixtures/cart.py]: Should this be hardcoded or do you have a way of making this dynamic or site/brand agnostic?

File: fixtures/cart.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+    "language": "en-US",
     "channel": "web",
     "currencyCode": "USD",
```
Comment (OSeraph) [inline fixtures/cart.py]: for `currencyCode` Are you planning on running theses tests for different currency types? 
If not we should discuss those scenarios, I think that might change the results.

File: fixtures/cart.py (hunk 1 above)
Comment (danggrianto) [reply fixtures/cart.py]: yeah doesn't really matter in the code. because this value will be passed anyway

File: fixtures/cart.py (hunk 2 above)
Comment (danggrianto) [reply fixtures/cart.py]: at this moment, there is no real code handling those, similar with location/region, there will be ticket handling region and there will be more tests regarding that.

Comment (danggrianto) [reply fixtures/cart.py]: as you can see as the tests progress, so does the fixtures.

**Pull request merged**